# Request part (empty if all we need is to "ask" for position)
---

# Response part
float64 x
float64 y
float64 z

